phantom-dsl/src/test/scala/com/websudos/phantom/builder/query/db/ordering/TimeSeriesTest.scala

```diff
@@ -29,17 +29,23 @@
  */
 package com.websudos.phantom.builder.query.db.ordering
 
+import com.datastax.driver.core.Session
+import com.twitter.util.{Future => TwitterFuture}
 import com.websudos.phantom.PhantomSuite
-
-import scala.concurrent.duration._
-
-import org.scalatest.concurrent.PatienceConfiguration
-
+import com.websudos.phantom.batch.BatchQuery
+import com.websudos.phantom.builder.Unspecified
+import com.websudos.phantom.builder.query.prepared._
+import com.websudos.phantom.connectors.KeySpace
 import com.websudos.phantom.dsl._
 import com.websudos.phantom.tables._
 import com.websudos.util.testing._
+import org.scalatest.concurrent.PatienceConfiguration
+
+import scala.concurrent.duration._
+import scala.concurrent.{Future => ScalaFuture}
 
 class TimeSeriesTest extends PhantomSuite {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Imports should be grouped together](https://www.codacy.com/app/flavian/phantom_2/file/2242677072/issues/source?bid=2847375&fileBranchId=3179387#l48)